test/e2e/storage/subpath.go

```diff
@@ -718,6 +785,15 @@ func (s *gcepdSource) createVolume(f *framework.Framework) volInfo {
 	}
 }
 
+func (s *gcepdSource) readOnlyVolume() *v1.VolumeSource {
+	return &v1.VolumeSource{
+		PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
```

Comment:
should this be GCEPDSource? Or maybe this should have a gcepdPVCSource type receiver. Or maybe the naming is just inconsistent

Reply:
ack